### PR TITLE
Test/car router

### PR DESCRIPTION
**Pull request by ohjonah**
https://github.com/nickjaz/parkify/issues/31

### CODE REVIEW

File: test/car/car-delete-test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+const User = require('../../model/user.js');
+const Car = require('../../model/car.js');
+const generateUser = require('../lib/generate-user.js');
+const generateCar = require('../lib/generate-car.js');
+
+const url = `http://localhost:${process.env.PORT}`;
+
+require('../../server.js');
+
+describe('GET: /api/car/:id', function() {
```
Comment (nharren) [inline test/car/car-delete-test.js]: This should be 'DELETE'.

**Review by nharren — COMMENTED** [review]
Can you rename your tests to match the format on the wiki?

File: test/car/car-delete-test.js (hunk 1 above)
Comment (ohjonah) [reply test/car/car-delete-test.js]: Good catch! Fixed!

**Review by sharmarkei — DISMISSED** [review]

File: test/car/put-test.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 'use strict';
+// Hi Nathan
```
Comment (sharmarkei) [inline test/car/put-test.js]: delete this please

File: test/car/put-test.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
+'use strict';
+// Hi Nathan
```
Comment (sharmarkei) [inline test/car/put-test.js]: delete this please

**Review by sharmarkei — APPROVED** [review]

**Pull request merged**